src/i18n/pt-br/nav.ts

```diff
@@ -1,75 +1,78 @@
-	'guides/integrations-guide': 'Adicione integrações',
-	'guides/deploy': 'Faça deploy do seu site',
-	'guides/recipes': 'Mais Receitas',
+	'guides/rss': 'Adicione um feed RSS',
 
-	features: 'Guias',
-	'guides/framework-components': 'Frameworks de UI',
+	routes: 'Rotas e Navegação',
 	'guides/routing': 'Roteamento',
-	'guides/markdown-content': 'Markdown e MDX',
-	'guides/content-collections': 'Coleções de Conteúdo',
-	'guides/client-side-scripts': 'Scripts & Manipulação de Eventos',
+	'guides/endpoints': 'Endpoints',
+	'guides/prefetch': 'Pré-carregamento',
+	'guides/middleware': 'Middleware',
+	'guides/internationalization': 'Internacionalização',
+	'guides/view-transitions': 'Transições de Visualização',
+
+	assets: 'Recursos',
 	'guides/styling': 'Estilização e CSS',
 	'guides/images': 'Imagens',
 	'guides/fonts': 'Fontes',
-	'guides/imports': 'Importações',
-	'guides/server-side-rendering': 'Adaptadores SSR',
-	'guides/astro-db': 'Astro DB',
-	'guides/endpoints': 'Endpoints',
+
+	data: 'Conecte seus Dados',
 	'guides/data-fetching': 'Busca de Dados',
-	'guides/middleware': 'Middleware',
+	'guides/astro-db': 'Astro DB',
+	'guides/backend': 'Adicione serviços de backend',
 	'guides/ecommerce': 'Comércio Eletrônico',
-	'guides/testing': 'Testes',
-	'guides/view-transitions': 'Transições de Visualização',
-	'guides/prefetch': 'Pré-carregamento',
-	'guides/troubleshooting': 'Solução de Problemas',
 	'guides/authentication': 'Autenticação',
-
-	configuration: 'Configuração',
-	'guides/configuring-astro': 'O Arquivo de Configuração Astro',
-	'guides/typescript': 'TypeScript',
-	'guides/aliases': 'Atalhos de Importação',
 	'guides/environment-variables': 'Variáveis de Ambiente',
 
+	clientSide: 'Interatividade no Lado do Cliente',
+	'guides/framework-components': 'Frameworks de UI',
+	'guides/client-side-scripts': 'Scripts & Manipulação de Eventos',
+
+	maintain: 'Mantendo seu Projeto',
+	'upgrade-astro': 'Atualize o Astro',
+	'guides/testing': 'Testes',
+	'guides/troubleshooting': 'Solução de Problemas',
+
 	reference: 'Referência',
 	'reference/configuration-reference': 'Configuração',
 	'reference/api-reference': 'API de Runtime',
+	'reference/cli-reference': 'A CLI do Astro',
+	'reference/directives-reference': 'Diretivas de Template',
+	'guides/typescript': 'TypeScript',
+	'reference/error-reference': 'Referência de Erros',
+
+	dev: 'Outras APIs de desenvolvimento',
 	'reference/integrations-reference': 'API de Integrações',
 	'reference/adapter-reference': 'API de Adaptadores',
 	'reference/image-service-reference': 'API de Serviço de Imagem',
-	'reference/directives-reference': 'Diretivas de Template',
-	'reference/cli-reference': 'A CLI do Astro',
-	'reference/error-reference': 'Referência de Erros',
-	'guides/publish-to-npm': 'Publique no NPM',
+	'reference/dev-toolbar-app-reference': 'Barra de Ferramentas de Desenvolvimento',
+	'reference/container-reference': 'API Container (Experimental)',
```

Comment:
nit: nós traduzimos a descrição de outras APIs, para manter a consistência vale a pena traduzir essa também.
```suggestion
	'reference/container-reference': 'API de Contêiner (Experimental)',
```